docs/source/examples/workspaces/openstack-mock-workspace/PinFile

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Why do we need a new workspace for this?

Reply:
@14rcole I just wanted to keep the mocked workspaces different from the actual ones. 
Since the linchpin.conf is different and mock data is only applicable to specific examples. 
for the time being.

Reply:
I thought that we were going to mock all of the providers?

Regardless, the workspaces need to be provisionable in both a mocked and non-mocked way.  Otherwise we can't effectively verify that the data being sent back from a given provider doesn't change.  Why don't we have `linchpin.conf` and `linchpin-mock.conf` in `docs/source/examples/workspaces` then have the integration tests look for a flag that, when set, uses the mock config/data instead of the normal data.

Reply:
@14rcole Yes we will be mocking all providers. 
It takes time to get all the sample data from all the providers. 
Having two linchpin.conf makes sense. 
I think we have option in linchpin to specify the linchpin.conf through the command line. 
we can make use of it. 
I will try to implement the same and maintain one workspace for both.